casadi-3/include/highs/filereaderlp/reader.hpp

```diff
@@ -1,10 +1,10 @@
-#ifndef __READERLP_READER_HPP__

```

Comment:
I am using Mac OS so my downloaded casadi may be different from yours. What OS do you use? Are the changes due to your downloaded casadi for your OS? If we merge your change, will the code broken for Mac OS ?

If we want the code to run for multiple OSs, I think the best way is for you to create a different casadi folder for example named "casadi_for_win" and upload all.

Reply:
Okay. I will do that.